.fdd-adapter/specs/domain-model.md

```
@@ -0,0 +1,219 @@
+#[derive(Debug, Clone, Serialize, Deserialize, ToSchema)]
+#[serde(rename_all = "snake_case")]
+pub enum ModuleStatus {
+    Active,
+    Inactive,
+    Error,
+}
+```
+
+## GTS Validation (MANDATORY)
+
+**Tool**: `gts` CLI from https://github.com/GlobalTypeSystem/gts-rust
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)